### PR TITLE
Pull request for E2E DETS

### PR DESCRIPTION
**Pull request by amoldashwant**
This pull request includes  working End-to-end data exchange test service initial source code.

### CODE REVIEW

File: src/main/java/org/connector/e2etestservice/Utils.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.connector.e2etestservice;
```
Comment (sachinargade123) [inline src/main/java/org/connector/e2etestservice/Utils.java:22]: Please add copy right header

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ - Spring boot version update
+ - UI refactoring
+ - Test connector as a provider flow added
+ - Response variable name change
+ - UI page for testing connector
+ - Addition of helm charts
+ - Added AUTHORS.md, CHANGELOG.md, CODE_OF_CONDUCT.md, CONTRIBUTING.md, INSTALL.md, NOTICE.md, SECURITY.md file
+ - Added LICENSE file
+ - Open api changes
+ - Fetch preconfgiured connector url from properties file
```
Comment (sachinargade123) [inline CHANGELOG.md]: @amoldashwant correct typos

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ - Demo UI update and dependency issues resolved
+ - Testconnector api changes for autosetup email support
```
Comment (sachinargade123) [inline CHANGELOG.md]: @amoldashwant correct typo

**Review by sachinargade123 — CHANGES_REQUESTED** [review]
@amoldashwant Please correct typo, add missing copyright header

File: src/main/java/org/connector/e2etestservice/Utils.java (hunk 1 above)
Comment (amoldashwant) [reply src/main/java/org/connector/e2etestservice/Utils.java:22]: Changes done

File: CHANGELOG.md (hunk 2 above)
Comment (amoldashwant) [reply CHANGELOG.md]: Changes done

File: CHANGELOG.md (hunk 3 above)
Comment (amoldashwant) [reply CHANGELOG.md]: Changes done

**Comment by Siegfriedk** [comment]
Will merge it and will open an initial contribution for it.

**Pull request merged**